dnsdist packing on openSUSE Build Serviece fails if *not* built --with-protobuf

I guess it's because the undefined return value in protobuf.cc:155

```
[  241s] Preparing packages...
[  242s] dnsdist-1.0~git.gb81a85f-6.1.x86_64
[  242s] dnsdist-debuginfo-1.0~git.gb81a85f-6.1.x86_64
[  243s] dnsdist-debugsource-1.0~git.gb81a85f-6.1.x86_64
[  243s] ... running 01-check-debuginfo
[  243s] ... testing for empty debuginfo packages
[  243s] ... running 02-check-gcc-output
[  243s] ... testing for serious compiler warnings
[  243s]     (using /usr/lib/build/checks-data/check_gcc_output)
[  243s]     (using //.build.log)
[  243s] 
[  243s] I: Program returns random data in a function
[  243s] E: dnsdist no-return-in-nonvoid-function protobuf.cc:155
[  243s] 
[  243s] I: Program returns random data in a function
[  243s] E: dnsdist no-return-in-nonvoid-function protobuf.cc:155
[  243s] 
[  243s] cloud133 failed "build dnsdist.spec" at Thu Jul 28 08:35:34 UTC 2016.
```

GCC indeed gives a compiler warning (when configuring with `--without-protobuf`)

```
protobuf.cc: In member function ‘std::__cxx11::string DNSProtoBufMessage::toDebugString() const’:
protobuf.cc:155:1: warning: no return statement in function returning non-void [-Wreturn-type]
 }
 ^
```

And this warning turns into an error I think

--- a/pdns/protobuf.cc
+++ b/pdns/protobuf.cc
@@ -151,6 +151,8 @@ std::string DNSProtoBufMessage::toDebugString() const
 {
 #ifdef HAVE_PROTOBUF
   return d_message.DebugString();
+#else
+  return std::string();
 #endif /* HAVE_PROTOBUF */
 }
 

